envi/symstore/resolver.py

```diff
@@ -201,7 +201,7 @@ def delSymbol(self, sym):
             subres = self.symnames.get(sym.fname)
 
         # Potentially del it from the sub resolver's namespace
-        if subres is not None:
+        if subres is not None and not isinstance(subres, tuple):
```

Comment:
because we store sub-resolvers in symnames, not just symbols.  without this, we're ending up with trying to use a tuple (symbol) as a sub-resolver.

this may be a great start to symbol management, but it requires a bit of love, more than can be done in either of these two PR's i'm working on.